public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
+import { useLocation } from 'react-router-dom';
 import { cloneDeep } from 'lodash';
 import { css } from '@emotion/css';
 import { GrafanaTheme2, NavModelItem } from '@grafana/data';
 import { locationService } from '@grafana/runtime';
 import { Icon, IconName, styleMixins, useTheme2 } from '@grafana/ui';
 import config from '../../config';
+import { linkIsActive } from './utils';
 import SideMenuItem from './SideMenuItem';
 
 const TopSection = () => {
+  const location = useLocation();
+  const query = new URLSearchParams(location.search);
+  const isSearchActive = query.get('search') === 'open';
```

Comment:
Might want to consider wrapping this logic up into a custom hook or utility function?